example/_posts/2018-11-7-frontmatter-in-vuepress-3.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: frontmatter in vuepress 3
```

Comment:
3 ? ^^

Reply:
This directory is  just an example for testing or 
demonstrating.

Reply:
i get it but it could be great to not create confusion

Reply:
Take a look at the [example directory](https://github.com/vuepressjs/vuepress-theme-blog/tree/master/example/_posts) and you'll find that they're just some dummy copied files for demonstrating. Each post got a number just for identifying.